store/job_storer.go

```diff
@@ -0,0 +1,43 @@
+package store
+
+import (
+	"context"
+	models "github.com/ONSdigital/dp-search-reindex-api/models"
+	"github.com/ONSdigital/log.go/log"
+	"time"
+)
+
+type JobStorer struct {
+	JobsMap map[string]models.Job
+}
+
+// CreateJob creates a new Job resource and stores it in the JobsMap
+func (js *JobStorer) CreateJob(ctx context.Context, id string) (models.Job, error) {
+
+	log.Event(ctx, "creating job", log.Data{"id": id})
+	newJob := models.Job{
```

Comment:
Consider creating a `NewJob(id string)` func in the models package - it removes the burden on the caller to init a job object with the default values.

Reply:
very good advice - done